packages/expo-cli/src/commands/eas-build/utils/expoUpdates.ts

```diff
@@ -0,0 +1,370 @@
+import { AndroidConfig, ExpoConfig, getConfig, getPackageJson, IOSConfig } from '@expo/config';
+import plist from '@expo/plist';
+import { UserManager } from '@expo/xdl';
+import chalk from 'chalk';
+import figures from 'figures';
+import * as fs from 'fs-extra';
+import glob from 'glob';
+import ora from 'ora';
+import path from 'path';
+import xcode from 'xcode';
+
+import { gitAddAsync } from '../../../git';
+import log from '../../../log';
+import * as gitUtils from './git';
+
+const iOSBuildScript = '../node_modules/expo-updates/scripts/create-manifest-ios.sh';
+const androidBuildScript =
+  'apply from: "../../node_modules/expo-updates/scripts/create-manifest-android.gradle"';
+
+export async function isUpdatesConfiguredAsync(projectDir: string): Promise<boolean> {
+  if (!isExpoUpdatesInstalled(projectDir)) {
+    return true;
+  }
+
+  const { exp, username } = await getConfigurationOptionsAsync(projectDir);
+
+  return (
+    (await isUpdatesConfiguredAndroidAsync(projectDir, exp, username)) &&
+    (await isUpdatesConfiguredIOS(projectDir, exp, username))
+  );
+}
+
+export async function configureUpdatesAsync({
+  projectDir,
+  nonInteractive,
+}: {
+  projectDir: string;
+  nonInteractive: boolean;
+}) {
+  if (!isExpoUpdatesInstalled(projectDir)) {
+    return;
+  }
+
+  const spinner = ora('Configuring expo-updates');
+
+  try {
+    const { exp, username } = await getConfigurationOptionsAsync(projectDir);
+
+    await configureUpdatesAndroidAsync(projectDir, exp, username);
+    await configureUpdatesIOSAsync(projectDir, exp, username);
+
+    await gitUtils.ensureGitStatusIsCleanAsync();
+
+    spinner.succeed();
+  } catch (err) {
+    if (err instanceof gitUtils.DirtyGitTreeError) {
+      spinner.succeed(`We configured expo-updates in your project `);
+      log.newLine();
+
+      try {
+        await gitUtils.reviewAndCommitChangesAsync(`Configure expo-updates`, { nonInteractive });
+
+        log(`${chalk.green(figures.tick)} Successfully committed the configuration changes.`);
+      } catch (e) {
+        throw new Error(
+          "Aborting, run the command again once you're ready. Make sure to commit any changes you've made."
+        );
+      }
+    } else {
+      spinner.fail();
+      throw err;
+    }
+  }
+}
+
+async function getConfigurationOptionsAsync(
+  projectDir: string
+): Promise<{ exp: ExpoConfig; username: string | null }> {
+  const username = await UserManager.getCurrentUsernameAsync();
+
+  const { exp } = getConfig(projectDir);
```

Comment:
```suggestion
  const { exp } = getConfig(projectDir, { skipSDKVersionRequirement: true });
```
Otherwise there will be an error that would prevent the case where `runtimeVersion` is defined but `sdkVersion` isn't. Not sure if this is the intended functionality or not tho.